### PR TITLE
dev/reference/*: add Check the TiDB Cluster Status Using SQL Statements and update Information Schema

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

- Add a doc of SQL Diagnostic Kit
- Add table info in Information Schema

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1445/

**Note:** TOC.md is not updated because the title of the SQL Diagnostic Kit doc is not determined.

### Which version does your change affect? 

dev & v3.0

Once the content in the dev repo is fixed, I will replicate it to the v3.0 repo.

@kolbe @winoros PTAL

### CODE REVIEW

File: dev/reference/system-databases/information-schema.md [hunk 1]
```diff
@@ -12,7 +13,7 @@ As part of MySQL compatibility, TiDB supports a number of `INFORMATION_SCHEMA` t
 
 ### CHARACTER\_SETS table
 
-The `CHARACTER_SETS` table provides information about [character sets](/reference/sql/character-set.md). The default character set in TiDB is `utf8mb4`. Additional character sets in this table are included for compatibility with MySQL:
+The `CHARACTER_SETS` table provides information about [character sets](/dev/reference/sql/character-set.md). The default character set in TiDB is `utf8mb4`. Additional character sets in this table are included for compatibility with MySQL:
```
Comment (morgo) [inline dev/reference/system-databases/information-schema.md]:
```suggestion
The `CHARACTER_SETS` table provides information about [character sets](/reference/sql/character-set.md). The default character set in TiDB is `utf8mb4`. Additional character sets in this table are included for compatibility with MySQL:
```

**Comment by CaitinChen** [comment]
@morgo Yeah, I will update the content in the v3.0 repo when that in the dev repo is fixed, as shown in the PR description.

File: dev/reference/system-databases/information-schema.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: Information Schema
 summary: Learn how to use Information Schema in TiDB.
 category: reference
+aliases: ['/docs/sql/information-schema/']
```
Comment (morgo) [inline dev/reference/system-databases/information-schema.md]: Be sure to remove this line from the `/dev` version.

Comment (CaitinChen) [reply dev/reference/system-databases/information-schema.md]: @morgo Thanks!

File: dev/reference/performance/check-cluster-status-using-sql-statements.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Check the TiDB Cluter Status Using SQL Statements
```
Comment (kolbe) [inline dev/reference/performance/check-cluster-status-using-sql-statements.md]:
```suggestion
title: Check the TiDB Cluster Status Using SQL Statements
```

File: dev/reference/performance/check-cluster-status-using-sql-statements.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+summary: This document introduces that TiDB offers some SQL statements and system tables to check the TiDB cluster status.
+category: reference
+---
+
+# Check the TiDB Cluter Status Using SQL Statements
```
Comment (kolbe) [inline dev/reference/performance/check-cluster-status-using-sql-statements.md]:
```suggestion
# Check the TiDB Cluster Status Using SQL Statements
```

**Comment by CaitinChen** [comment]
@kolbe Thanks! PTAL again.

**Pull request merged**